app/controllers/api/v1/rents_controller.rb

```diff
@@ -2,6 +2,7 @@ module Api
   module V1
     class RentsController < ApiController
       before_action :authenticate_user!
+      before_action :set_locale
```

Comment:
what about putting this in the ApiController so every controller could use this logic?